slasher/rpc/detect_update_min_max_span.go

```diff
@@ -45,20 +44,20 @@ func detectMin(attestationEpochSpan uint64,
 // target epoch that has a lower (earlier) source epoch.
 // Logic for this detection method was designed by https://github.com/protolambda
 // Detailed here: https://github.com/protolambda/eth2-surround/blob/master/README.md#min-max-surround
-func (ss *Server) DetectAndUpdateMaxEpochSpan(ctx context.Context, source uint64, target uint64, validatorIdx uint64) (uint64, error) {
+func (ss *Server) DetectAndUpdateMaxEpochSpan(ctx context.Context, source uint64, target uint64, validatorIdx uint64, spanMap *ethpb.EpochSpanMap) (uint64, *ethpb.EpochSpanMap, error) {
```

Comment:
This line is too long, can we break it down into multiple lines?

Reply:
addressed in #4230